structs/vm.go

```diff
@@ -39,3 +41,33 @@ func (this *VM) PingDocker() bool {
 
     return err == nil && response.StatusCode == 200
 }
+
+func (this *VM) GetDockerVersion() (string, error) {
+    dockerAddress := fmt.Sprintf("http://%s:%s/%s/version",
+        this.Address, this.Port, this.Version)
+
+    req, err := http.NewRequest("GET", dockerAddress, nil)
+    if err != nil {
+        return "v1", err
```

Comment:
@theMagicalKarp this is a nitpick I just thought of, but do you think it'd be better to return `this.Version` here as a default?  In case `v1` is ever not our default version for future stuff?